static/src/javascripts/components/analytics/analytics.js

```diff
@@ -182,39 +181,71 @@ try {
-        *@
+        var userFromCookie = window.guardian.config.user;
 
-        s.t();
+        if (userFromCookie) {
+            s.prop2 = 'GUID:' + userFromCookie.id;
+            s.eVar2 = 'GUID:' + userFromCookie.id;
+        }
 
+        s.prop31    = userFromCookie ? 'registered user' : 'guest user';
+        s.eVar31    = userFromCookie ? 'registered user' : 'guest user';
+
+        s.prop40    = window.guardian.adBlockers.generic || window.guardian.adBlockers.ffAdblockPlus;
+
+        try {
+            var navInteractionData = window.sessionStorage.getItem(NG_STORAGE_KEY);
+
+            var navInteraction = JSON.parse(navInteractionData);
+
+            if (navInteraction) {
+                trackNavigationInteraction(navInteraction.value);
+                window.sessionStorage.removeItem(R2_STORAGE_KEY);
+                window.sessionStorage.removeItem(NG_STORAGE_KEY);
+            }
+
+        } catch (e) { }
+
+        // Sponsored content
+        if(window.guardian.isModernBrowser) {
+            var contentNodes = document.querySelectorAll('[data-sponsorship]');
+            var sponsoredContentArray = Array.prototype.slice.call(contentNodes);
+            s.prop38 = getSponsoredContentTrackingData(sponsoredContentArray);
+        }
+
+        /*
+         this makes the call to Omniture.
+         `s.t()` records a page view so should only be called once
+         */
+
+        s.t();
 
         var checkForPageViewInterval = setInterval(function () {
-            @*
-                s_i_guardiangu-network is a globally defined Image() object created by Omniture
-                It does not sit in the DOM tree, and seems to be the only surefire way
-                to check if the intial beacon has been successfully sent
-            *@
+            /*
+             s_i_guardiangu-network is a globally defined Image() object created by Omniture
+             It does not sit in the DOM tree, and seems to be the only surefire way
+             to check if the intial beacon has been successfully sent
+             */
             var img = window['s_i_' + window.s_account.split(',').join('_')];
             if (typeof (img) !== 'undefined' && (img.complete === true || img.width + img.height > 0)) {
                 clearInterval(checkForPageViewInterval);
 
                 var pageView = new Image();
-                pageView.src = "@{Configuration.debug.beaconUrl}/count/pva.gif";
+                pageView.src = guardian.config.page.beaconUrl + '/count/pva.gif';
+
             }
         }, 100);
 
         // Give up after 10 seconds
         setTimeout(function () {
             clearInterval(checkForPageViewInterval);
         }, 10000);
-
-
-    })();
-
-} catch(e) {
-    (new Image()).src = '@{Configuration.debug.beaconUrl}/count/omniture-pageview-error.gif';
-}
-
-
+    } catch(e) {
+        (new Image()).src = guardian.config.page.beaconUrl + '/count/omniture-pageview-error.gif';
+    }
+
+    return {
```

Comment:
If these are exclusively for testing, it might be worth identifying them as such. I don't think there is a standard approach, some return a test object that reveal the functions](https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/utils/cookies.js#L108), others use `_funcName`. @regiskuckaertz did we ever decide on an approach to revealing test-only methods?